packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -311,7 +311,8 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 					`ss__facet--${facet.field}`,
 					`${facet.collapsed ? 'ss__facet--collapsed' : ''}`,
 					className,
-					`${facet.display ? `ss__facet--${facet.display}` : ''}`
+					`${facet.display ? `ss__facet--${facet.display}` : ''}`,
+					((facet as ValueFacet)?.overflow?.remaining || 0) > 0 ? '' : 'ss__facet--showing-all'
```

Comment:
Not really hurting anything... but perhaps this new class shouldn't apply to facets that do not have this ability?
<img width="935" alt="image" src="https://github.com/user-attachments/assets/0b7078d2-2579-41f3-b789-77cead85fdde" />